python/pylib/data_sets.py

```diff
@@ -90,7 +90,7 @@
 DID_DUAL_IMU_RAW_MAG            = 84
 DID_DUAL_IMU_MAG                = 85
 DID_PREINTEGRATED_IMU_MAG       = 86
-DID_WHEEL_CONFIG                = 87
+DID_GROUND_VEHICLE                = 87
```

Comment:
Is wheel_config now a part of ground_vehicle?

Reply:
Yes.